src/app/demo.component.css

```diff
@@ -0,0 +1,6 @@
+host {
```

Comment:
I would prefer to use a class of `.app-demo` for the demo component.

Reply:
Whoops! I've added back the package-lock.json! Also updated the demo.component.css, please review!